knowledge-base/unswipe-item-collectionview.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Programmatically Unswiping an Item in CollectionView
+description: Learn how to unswipe an item in CollectionView for .NET MAUI programmatically using the EndItemSwipe method.
+type: how-to
+page_title: How to Unswipe an Item Programmatically in RadCollectionView for .NET MAUI
+slug: unswipe-item-collectionview
+tags: collectionview, .net maui, unswipe, item, programmatically
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 10.1.0 | Telerik UI for .NET MAUI CollectionView | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova) | 
+
+## Description
+
+In scenarios where an item in the RadCollectionView is swiped, there might be a need to programmatically return it to its original position. 
```

Comment:
You may want to use "CollectionView" on a few occasions in order to introduce an additional search term.
```suggestion
In scenarios where an item in the CollectionView is swiped, there might be a need to programmatically return it to its original position. 
```